.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -1,13 +1,11 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
+
+name: Feature
+about: You have an idea for a new capability or a refinement to an existing one
 title: ''
 labels: type/feature
 assignees: ''
-
----
-
-**Is your feature request related to a problem? Please describe.**
+---**Is your feature request related to a problem? Please describe.**
```

Comment:
Hopefully github can properly parse it, the `---` should probably be before the title as it was before. The rendered version is broken